### PR TITLE
UIUX Update

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

This was one of my requests on the last mass UIUX update, I deferred it so we could merge. This is breaking down the overlong "bias in user research" entry into the concept and a number of terms.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/uiux/concepts/bias-in-user-research/terms/interview-bias/interview-bias.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Interview bias** refers to bias due to a participant's perception of the interviewer. This may be due to a user's own biased perception of an interviewer's ethnicity, age, gender, or physical appearance &mdash; or even details like body language or facial expressions. Interview bias can be difficult to avoid, so it is just important to keep in mind when evaluating the data.
```
Comment (ishg-153) [inline content/uiux/concepts/bias-in-user-research/terms/interview-bias/interview-bias.md:14]: I think this entry is supposed to be for Interviewer Bias.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
Hey @SSwiniarski. I have suggested some changes, please let me know if I misunderstood anything.

For the framing effect entry, can sub it for cognitive bias instead? Since the framing effect stems from Cognitive bias.

**Comment by SSwiniarski** [comment]
> Hey @SSwiniarski. I have suggested some changes, please let me know if I misunderstood anything.
> 
> For the framing effect entry, can we sub it for cognitive bias instead? Since the framing effect stems from Cognitive bias.

@ishg-153  I'd rather cognitive bias be done in it's own PR. Other than that, I addressed your comments.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Comment by SSwiniarski** [comment]
@dakshdeepHERE, committed your suggestions.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good

**Pull request merged**